src/modules/pece/pece_security/pece_security.features.defaultconfig.inc

```diff
@@ -0,0 +1,212 @@
+  $strongarm->name = 'login_security_notice_attempts_available';
+  $strongarm->value = 0;
+  $export['login_security_notice_attempts_available'] = $strongarm;
+
+  $strongarm = new stdClass();
+  $strongarm->disabled = FALSE; /* Edit this to true to make a default strongarm disabled initially */
+  $strongarm->api_version = 1;
+  $strongarm->name = 'login_security_notice_attempts_message';
+  $strongarm->value = 'You have used @user_current_count out of @user_block_attempts login attempts. After all @user_block_attempts have been used, you will be unable to login.';
+  $export['login_security_notice_attempts_message'] = $strongarm;
+
+  $strongarm = new stdClass();
+  $strongarm->disabled = FALSE; /* Edit this to true to make a default strongarm disabled initially */
+  $strongarm->api_version = 1;
+  $strongarm->name = 'login_security_track_time';
+  $strongarm->value = '24';
+  $export['login_security_track_time'] = $strongarm;
+
+  $strongarm = new stdClass();
+  $strongarm->disabled = FALSE; /* Edit this to true to make a default strongarm disabled initially */
+  $strongarm->api_version = 1;
+  $strongarm->name = 'login_security_user_blocked';
+  $strongarm->value = 'The user @username has been blocked due to failed login attempts.';
+  $export['login_security_user_blocked'] = $strongarm;
+
+  $strongarm = new stdClass();
+  $strongarm->disabled = FALSE; /* Edit this to true to make a default strongarm disabled initially */
+  $strongarm->api_version = 1;
+  $strongarm->name = 'login_security_user_blocked_email_body';
+  $strongarm->value = 'The user @username (@edit_uri) has been blocked at @site due to the amount of failed login attempts. Please check the logs for more information.';
+  $export['login_security_user_blocked_email_body'] = $strongarm;
+
+  $strongarm = new stdClass();
+  $strongarm->disabled = FALSE; /* Edit this to true to make a default strongarm disabled initially */
+  $strongarm->api_version = 1;
+  $strongarm->name = 'login_security_user_blocked_email_subject';
+  $strongarm->value = 'Security action: The user @username has been blocked.';
+  $export['login_security_user_blocked_email_subject'] = $strongarm;
+
+  $strongarm = new stdClass();
+  $strongarm->disabled = FALSE; /* Edit this to true to make a default strongarm disabled initially */
+  $strongarm->api_version = 1;
+  $strongarm->name = 'login_security_user_blocked_email_user';
+  $strongarm->value = 'admin';
+  $export['login_security_user_blocked_email_user'] = $strongarm;
+
+  $strongarm = new stdClass();
+  $strongarm->disabled = FALSE; /* Edit this to true to make a default strongarm disabled initially */
+  $strongarm->api_version = 1;
+  $strongarm->name = 'login_security_user_wrong_count';
+  $strongarm->value = '0';
+  $export['login_security_user_wrong_count'] = $strongarm;
+
+  return $export;
+}
+
+/**
+ * Implements hook_defaultconfig_user_default_permissions().
+ */
+function pece_security_defaultconfig_user_default_permissions() {
```

Comment:
Hello @unixjazz, both permissions ("administer aes" and "view passwords") should be exported using Default Config so PECE administrators can override them from Drupal permissions administration, to fit their use case.